db/schema.rb

```diff
@@ -1,101 +1,96 @@
+    t.boolean "has_sample_count", default: false, null: false
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.string   "identifier_data_type", limit: 255, default: "alphanumeric", null: false
-    t.string   "labware_type",         limit: 255
+    t.string "identifier_data_type", default: "alphanumeric", null: false
+    t.string "labware_type"
   end
 
-  create_table "assets", id: :integer, force: :cascade do |t|
-    t.string   "identifier",              limit: 255,             null: false
-    t.integer  "asset_type_id",           limit: 4,               null: false
-    t.integer  "workflow_id",             limit: 4,               null: false
-    t.integer  "comment_id",              limit: 4
-    t.integer  "batch_id",                limit: 4,               null: false
-    t.string   "study",                   limit: 255
-    t.integer  "sample_count",            limit: 4,   default: 1, null: false
+  create_table "assets", id: :integer, charset: "utf8", collation: "utf8_unicode_ci", force: :cascade do |t|
+    t.string "identifier", null: false
+    t.integer "asset_type_id", null: false
+    t.integer "workflow_id", null: false
+    t.integer "comment_id"
+    t.integer "batch_id", null: false
+    t.string "study"
+    t.integer "sample_count", default: 1, null: false
     t.datetime "created_at"
     t.datetime "updated_at"
     t.datetime "completed_at"
     t.datetime "reported_at"
-    t.integer  "pipeline_destination_id", limit: 4
-    t.datetime "begun_at",                                        null: false
-    t.integer  "cost_code_id",            limit: 4
-    t.string   "project",                 limit: 255
+    t.integer "pipeline_destination_id"
+    t.datetime "begun_at", null: false
+    t.integer "cost_code_id"
+    t.string "project"
+    t.index ["asset_type_id"], name: "fk_assets_to_asset_types"
+    t.index ["batch_id"], name: "index_assets_on_batch_id"
+    t.index ["comment_id"], name: "fk_assets_to_comments"
+    t.index ["completed_at"], name: "index_assets_on_completed_at"
+    t.index ["identifier"], name: "index_assets_on_identifier"
+    t.index ["workflow_id"], name: "fk_assets_to_workflows"
   end
 
-  add_index "assets", ["asset_type_id"], name: "index_assets_on_asset_type_id", using: :btree
-  add_index "assets", ["batch_id"], name: "index_assets_on_batch_id", using: :btree
-  add_index "assets", ["comment_id"], name: "index_assets_on_comment_id", using: :btree
-  add_index "assets", ["identifier"], name: "index_assets_on_identifier", using: :btree
-  add_index "assets", ["workflow_id"], name: "index_assets_on_workflow_id", using: :btree
-
-  create_table "batches", id: :integer, force: :cascade do |t|
+  create_table "batches", id: :integer, charset: "utf8", collation: "utf8_unicode_ci", force: :cascade do |t|
     t.datetime "created_at"
     t.datetime "updated_at"
   end
 
-  create_table "comments", id: :integer, force: :cascade do |t|
-    t.text     "comment",    limit: 65535
```

Comment:
Another default limit?

Reply:
I think it must be. This one was a bit trickier to google...